extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -58,62 +42,108 @@ describes.realWin('Bind', {
       newElement.className = 'i-amphtml-foo -amp-foo amp-foo';
       newElement.mutatedAttributesCallback = () => {};
     }
-    const parent = env.win.document.getElementById('parent');
     parent.appendChild(newElement);
     return newElement;
-  }
+  };
+}
 
+function onBindReady_(bind, env) {
   /**
-   * Resolves when Bind service is fully initialized.
    * @return {!Promise}
    */
-  function onBindReady() {
+  return function() {
     return bind.initializePromiseForTesting().then(() => {
       env.flushVsync();
     });
-  }
+  };
+}
 
+function onBindReadyAndSetState_(bind, env) {
```

Comment:
I think these curried functions may be more readable if they were just created with just an `env`, and then require a `bind` to be passed in as the first arg. That way the tests that use two binds can be more  explicit and not depend on the curried function names for readability.